ui/src/components/credentials/CredentialLinkQR.tsx

```diff
@@ -230,7 +230,7 @@ export function CredentialLinkQR() {
   return (
     <CredentialQR
       qrCode={authQRCode.data.qrCode}
-      schemaType={authQRCode.data.linkDetail.schemaType}
+      schemaTypes={[authQRCode.data.linkDetail.schemaType]}
```

Comment:
Why does this one not have several, like issued potentially does?

Reply:
Can't tell. This is to be asked to backend. I suspect one of them is not correct on the backend side.

Reply:
Resolved on a call. Previous validation for a `tuple` (array of one fixed element) is more correct in this case. An array of none or multiple arrays is possible for a user DID, but not for one credential ID. It should never be `undefined`.

Reply:
Changed.